common/asyncworkflow/queue/provider/provider.go

```diff
@@ -38,6 +38,11 @@ type (
 		MetricsClient metrics.Client
 	}
 
+	QueueID struct {
```

Comment:
Can you add comments on this struct and fields to explain what they are?

Reply:
I accidentally committed this change. I'll remove this